doc/releases/release-notes-2.2.rst

```diff
@@ -179,7 +179,18 @@ Boards & SoC Support
 
 .. rst-class:: rst-columns
 
-   * <TBD>
+   * Atmel SAM4E
+   * Atmel SAMV71
+   * Broadcom BCM58400
+   * ST STM32F098xx
```

Comment:
Should `STM32F098xx` relocate to live with its friends?

Reply:
yeap, will fix this

Reply:
fixed now